tests/layers/attention/matrix_attention_test.py

```diff
@@ -6,6 +6,7 @@
 
 from deep_qa.layers.attention.matrix_attention import MatrixAttention
 from deep_qa.layers.wrappers.output_mask import OutputMask
+from deep_qa.common.params import Params
```

Comment:
We should probably just leave out `Params` for all of these layer tests, because we're not using `Params` for layers, anyway.  It's good for the rest of the tests, though.